src/Workspaces/Core/MSBuild.BuildHost/Rpc/Request.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.CodeAnalysis.Rpc
```

Comment:
nit: this is a very vague/general namespace.  this looks to be specific to MSBuild, so i would love if that was in the namespace somehow.

Reply:
Sure, can change.